ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vlan-interfaces.j2

```diff
@@ -50,6 +50,9 @@
 {%         set row_acl_out = vlan_interface.access_group_out | arista.avd.default('-') %}
 | {{ vlan_interface.name }} |  {{ row_vrf }}  |  {{ row_ip_addr }}  |  {{ row_ip_vaddr }}  |  {{ row_varp }}  |  {{ row_vrrp }}  |  {{ row_acl_in }}  |  {{ row_acl_out }}  |
 {%     endfor %}
+{# IP NAT Documentation #}
+{%     set ip_nat_interfaces = vlan_interfaces | arista.avd.default({}) %}
```

Comment:
```suggestion
{%     set ip_nat_interfaces = vlan_interfaces | arista.avd.default([]) %}
```

Reply:
Validation if `vlan_interfaces` is defined is already done in line 2. Thus defaulting is not required at all.